Server/src/app.module.ts

```diff
@@ -5,11 +5,19 @@ import { AppService } from './app.service';
 import { LocationModule } from './location/location.module';
 import config from 'ormconfig';
 import { Location } from './location/entities/location.entity';
+import { ConfigModule } from '@nestjs/config';
+import configuration from 'config/configuration';
 
```

Comment:
See comment on import grouping in #26, same here.

Reply:
This will be fix in PR #26